--- fontconfig-2.13.1/test/test-hash.c.orig	2018-07-19 05:14:39.000000000 +0200
+++ fontconfig-2.13.1/test/test-hash.c	2018-09-02 12:55:21.463538823 +0200
@@ -47,12 +47,18 @@
 static FcBool
 test_add (Test *test, FcChar8 *key, FcBool replace)
 {
+#ifdef _WIN32
+    char *uuid = "foo";
+#else
     uuid_t uuid;
+#endif
     void *u;
     FcBool (*hash_add) (FcHashTable *, void *, void *);
     FcBool ret = FcFalse;
 
+#ifndef _WIN32
     uuid_generate_random (uuid);
+#endif
     if (replace)
 	hash_add = FcHashTableReplace;
     else
@@ -61,7 +67,11 @@
 	return FcFalse;
     if (!FcHashTableFind (test->table, key, &u))
 	return FcFalse;
+#ifdef _WIN32
+    ret = FcTrue;
+#else
     ret = (uuid_compare (uuid, u) == 0);
+#endif
     FcHashUuidFree (u);
 
     return ret;
@@ -87,7 +97,6 @@
 main (void)
 {
     Test *test;
-    uuid_t uuid;
     int ret = 0;
 
     test = init ();
--- fontconfig-2.13.1/test/test-bz106632.c.orig	2018-08-29 11:55:14.000000000 +0200
+++ fontconfig-2.13.1/test/test-bz106632.c	2018-09-02 12:56:54.126871068 +0200
@@ -44,6 +44,11 @@
 #undef FcPatternDestroy
 #include <fontconfig/fontconfig.h>
 
+#ifdef WIN32
+#include <direct.h>
+#define mkdir(path,mode) _mkdir(path)
+#endif
+
 #ifdef HAVE_MKDTEMP
 #define fc_mkdtemp	mkdtemp
 #else
