--- fontconfig-2.11.1/src/fcint.h.orig	2014-01-20 09:14:20.000000000 +0100
+++ fontconfig-2.11.1/src/fcint.h	2014-03-25 20:02:08.436489743 +0100
@@ -29,6 +29,10 @@
 #include <config.h>
 #endif
 
+#ifdef _WIN32
+#include "fcwindows.h"
+#endif
+
 #include "fcstdint.h"
 
 #include <stdlib.h>
--- fontconfig-2.11.1/src/fcwindows.h.orig	2014-01-20 09:14:21.000000000 +0100
+++ fontconfig-2.11.1/src/fcwindows.h	2014-03-25 19:23:46.449919680 +0100
@@ -33,7 +33,7 @@
 
 #ifdef _WIN32
 #  ifndef _WIN32_WINNT
-#    define _WIN32_WINNT 0x0500
+#    define _WIN32_WINNT 0x0600
 #  endif
 #  define WIN32_LEAN_AND_MEAN
 #  define WIN32_EXTRA_LEAN
